strawberryfields/apps/graph/utils.py

```diff
@@ -153,3 +153,76 @@ def is_subgraph(subgraph: Iterable, graph: nx.Graph):
         return set(subgraph).issubset(graph.nodes)
     except TypeError:
         raise TypeError("subgraph and graph.nodes must be iterable")
+
+
+def is_clique(graph: nx.Graph) -> bool:
+    """Determines if the input graph is a clique
+
+    Args:
+        graph (nx.Graph): the input graph
+
+    Returns:
+        bool: returns ``True`` if input graph is a clique and ``False`` otherwise
+    """
+    edges = graph.edges
+    nodes = graph.order()
+
+    return len(edges) == nodes * (nodes - 1) / 2  # a clique has of n nodes has n*(n-1)/2 edges
```

Comment:
I would suggest removing this comment and making it part of the function docstring. Also, is it supposed to say "a clique has n nodes" ... ?

Reply:
Agree with @jswinarton that this might as well be in docstring.
I'd guess it should say "a clique of n nodes has n*(n-1)/2" edges.